app/models/need.rb

```diff
@@ -49,6 +49,10 @@ def self.total_children_served
 
   alias_attribute :duration, :expected_duration
 
+  def number_of_children
+    children.count
+  end
```

Comment:
Do we intend to use this somewhere outside of specs?

Reply:
Yes, this is used throughout the application. Currently, `number_of_children` is used to return the `number_of_children` property of a need. With this branch and the addition of children objects that belong to need adding this method allows the previous functionality to work like before but will instead return the total number of children objects instead of the legacy `number_of_children` property of the need.